Update documentation-government-developer-guide.md

mmydland opened the pull request:
The gallery URI for Fairfax seems to be incorrect.  The correct value is: https://gallery.azure.us

azurecla commented:
Hi __@PatAltimore__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (patricka). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Update documentation-government-developer-guide.md

The gallery URI for Fairfax seems to be incorrect.  The correct value is: https://gallery.azure.us
